lib/design_query.py

```diff
@@ -28,6 +28,7 @@
 from subprocess import Popen, PIPE, STDOUT
 from threading import Thread
 from queue import Queue, Empty
+import re
```

Comment:
This is an unused import probably added automatically by VSCode. Should be removed though.